frontend-next/src/credits.tsx

```diff
@@ -0,0 +1,233 @@
+import { Box, Heading, Text } from "@chakra-ui/react";
+
+export default function Credits() {
+    return (
+        <Box>
+            <Heading size="md">
+                Website Team <br />
+            </Heading>
+<br />
+            <Text>
+                Kyuubei - Web Design <br />
+                Jesse Zong - Frontend Website Designer <br />
+                Zapple - Backend Development & Frontend contributions <br />
```

Comment:
Does it not like the `&` here? Do you need to escape it?

Reply:
/shrug